scripts/get-collectd-plugins.py

```diff
@@ -0,0 +1,52 @@
+#!/bin/python
```

Comment:
Can the regular Linux build just use this now and delete the original "get plugins" script?